src/utils/RetailApiCursor.js

```diff
@@ -1,16 +1,13 @@
 const { Readable } = require('stream');
 const querystring = require('querystring');
 
-const { performRequest } = require('./HttpClient');
-
-class ApiCursor extends Readable {
-  constructor(baseUrl, resource, instance, qs, opts) {
+class RetailApiCursor extends Readable {
+  constructor(baseUrl, resource, instance, queryString) {
     super();
     this._baseUrl = baseUrl;
     this._resource = resource;
     this._instance = instance;
-    this._qs = qs;
-    this.opts = opts;
```

Comment:
Seems like opts was removed and `qs` was quite confusing so I gave the variable a better name.